### PR TITLE
Automatically build md from Pluto notebooks using PlutoStaticHTML. 

### PR DESCRIPTION
**Pull request by pwochner**
- Markdown files are built directly from Pluto notebooks. 
- Notebooks are executed before conversion => deprecated notebooks will fail.
- Remove obsolete html files.

### CODE REVIEW

File: docs/src/tutorials/advanced_search.jl [hunk 1]
```diff
@@ -139,7 +139,10 @@ problem_2 = Problem([IOExample(Dict(), x) for x ∈ 1:5])
 iterator_3 = BFSIterator(g_2, :Index, max_depth=2)
 
 # ╔═╡ 4821fd3a-ff2d-4991-99ad-76608d11b1da
+# ╠═╡ disabled = true
+#=╠═╡
 solution_3 = synth(problem_2, iterator_3)
+  ╠═╡ =#
```
Comment (ReubenJ) [inline docs/src/tutorials/advanced_search.jl:145]: Can we use a macro from the `Test` package to run this and test that an exception is thrown? Then the intended behavior is explicit in both text and code.

Comment (ReubenJ) [reply docs/src/tutorials/advanced_search.jl:145]: https://docs.julialang.org/en/v1/stdlib/Test/#Test.@test_throws

Comment (pwochner) [reply docs/src/tutorials/advanced_search.jl]: I added a test to the cell.

File: docs/src/tutorials/advanced_search.jl [hunk 2]
```diff
@@ -139,10 +142,7 @@ problem_2 = Problem([IOExample(Dict(), x) for x ∈ 1:5])
 iterator_3 = BFSIterator(g_2, :Index, max_depth=2)
 
 # ╔═╡ 4821fd3a-ff2d-4991-99ad-76608d11b1da
-# ╠═╡ disabled = true
-#=╠═╡
-solution_3 = synth(problem_2, iterator_3)
-  ╠═╡ =#
+Test.@test_throws HerbSearch.EvaluationError synth(problem_2, iterator_3)
```
Comment (ReubenJ) [inline docs/src/tutorials/advanced_search.jl:145]: Perfect! Looks good 🎉

**Review by ReubenJ — APPROVED** [review]

**Pull request merged**